### DISCUSSION TITLE
No Waybar loads at loging

### DISCUSSION THREAD

**Discussion by tinker-net** [Manjaro-Sway/manjaro-sway#574] [Q&A] [answered]
Hi to all.
I assume my question cant be a little bit silly, but i can't figure out a way to fix and maybe its really easy.
I have installed Manjaro Sway and i'm in love, but i can't make waybar start at loging and the only way is reload the config: mod+shift+c
The live session works like that to. 
I have installed the image you publish yesterday (but i have try a loooooot and everyone works like that)

Everything works fine, except plexamp (it seems a bug of the app behaviour with wayland and i don't know if i can do anything for now). But the fact the waybar does not load at login is really anoying. 

PS: also, i want to know how to edit files in superuser because the weather applet show me the weather in a place that is not where i live (and how can change it xD) 

thx you all. I will apreciate your help.

**Comment by boredland** [comment]
For the weather location, please check if you can override it like this: https://github.com/manjaro-sway/manjaro-sway/issues/284#issuecomment-1148510404

As for the hanging waybar, you could try if the current version from the AUR works better or if there are logs explaining what happens. Perhaps some script of ours is blocking the startup.

**Reply by tinker-net** [reply]
I have just reinstall because i mess around with nvidia drivers (my session is sway-unsupported gpu) because i want to activate the VA-API but i think i didn't achieve (or i don0t really know if i need it). 
after reinstall, everithing works as espected and the bar continue to show only when i reload the config. 

i will try the #284, but for the other part, i don't understand your comment. You say that i try to reinstall the waybar from AUR? it is a version less recent that the one i installed with the iso. So, what's the point? The logs, if you can tell me where they are, i post it with no problem, and the script, do you have any idea what can be? 

Anyways thank you for your answer ;0)

**Comment by tinker-net** [comment]
Hi again, 
I write to tell that i have definitively uninstall Manjaro Sway from my computer and i'm using i3. It is not the same, but it is quite similar. 
My problem is that I'm not able to configure properly the nouveau driver for my GPU (1050ti) and i think everything i do with Sway is a waste of time, cause my GPU is not gonna work properly. 
i try endeavour OS (Sway edition) and the VA API works well, so nouveau can be configurated and work with wayland and my GPU but i don `t know how to do it. 
It's a shame, i really love Manjaro Sway, but it is to complicated for me to make it work properly. 
Congrats for the distro i think it is the most beautiful, solid and fun to use i have try in Linux. 

Thx to you
Eduardo

**Reply by boredland** [reply]
Have you selected the "unsupported GPU" session in the greeter? At this point I am not even sure I understood the problem you've got.. :'(

**Reply by tinker-net** [reply]
Hi,

thanks for your answer, i really appreciate your support.

You're right, i have change my initial request and i'm meesing around. I make a little recap: 

My first request was about waybar: it does not load al login and only appears when i reload the sway config (i just reinstall the latest iso, from today, and it continues to work like that) in fact, when i run the live session it is the same and to install properly i must first close calamares, then mode+shift+c to reload the config, connect to internet and reload Calamares with sudo -E calamares, and then install. The waybar, still don't load at login and i don't understand why and what can be wrong. 

The second issue was about the weather in waybar, I haven't fix it (override doesn't work) but i have understand the way it works and i have the id of my city, so i will eventually fix it. 

But the third issue is the key. Using the OS i realise that the VA - API, so the hardware acceleration is not working. Even in unsopported GPU. I have a 1050ti (4gb) i u nderstand it is a very low, and a little bit old GPU but it works with EVERY other OS i have try to install in my computer (even, other distros with Sway, like Endeavour and Garuda) so, i understand it must be a way to make it work in Manjaro. Unfortunately, it is beyond my knowledge without any kind of support and it is something that i can ignore because i want a fully operational OS. This is why i prefer i3 over sway, even if i'm really in love with sway (it is the only desktop i like to use with any customization because it is useful and really beautiful) 

This is what has happend this past three days. 

I really admire your work and i think it is awesome, but i'm sad i can't make it work for me. 

Ps: i don't  understand why but Pamac can't update AUR repositories after enable it in the last iso (but it can be something related with my computer.

**Reply by tinker-net** [reply]
I must assume that Manjaro Sway does not support my GPU? 
i have try every method to  install the drivers (Pascal nvidia is supported by nouveau and it must work when I install it, or when theyre installed, whichis the case). In fact, i'm pretty sure that every problems i'm experiencing are something related to the iso, and not from my computer, because everything other OS and VMs (even others sway) i have tried works. 

So, this is the issue. 
thank you very much.

**Reply by boredland** [reply]
I don't think your missing GPU support is a sway issue. This surely isn't an issue about your GPU not being powerful enough. 

I have been using Manjaro sway on a Nvidia-GPU for quite some time with the proprietary drivers installed via `mhwd` and with sway started with the `unsupported-gpu` flag (which you need for Nvidia to work at all).

**Reply by boredland** [reply]
> don't understand why but Pamac

Can't help you there. I don't use pamac.

**Reply by boredland** [reply]
> My first request was about waybar

This might be related to the graphics drivers not working properly, but I think I just had this on a totally different setup too - will investigate.

**Reply by tinker-net** [reply]
Ok, the issue with Pamac is related with my inexperience with Arch. Thank you, i understand the way It works. Sorry for making silly questions 😃

**Reply by tinker-net** [reply]
I agree with what you say about waybar. In fact, i realised that the GPU was not working well trying to understand if the GPU was or not related with this issue.

**Reply by tinker-net** [reply]
![PXL_20230812_223243291](https://github.com/manjaro-sway/manjaro-sway/assets/141952222/4e2964a9-4ae3-4d46-825b-f3fbbbb0c3ee)

![PXL_20230812_140930293](https://github.com/manjaro-sway/manjaro-sway/assets/141952222/aec054a3-b6ca-41d0-9dcf-e59f00bf1876)
And, about the GPU, i understand web are agree: It is not a sway issue, and it is not related with my GPU, which normally should work (as It does un Garuda or Endeavour with nouveau). 
The workaround you mention, with de Nvidia propietary and unsuported GPU does not work At all. Directly Sway don't load. I put a shot about what It says. 
So my bet is there are something wrong with the iso, maybe my computer is too much complicated, but i think the problem is on the iso and related with Wayland not loading (or putting on) the VA-API. 
I know my vocabulary is not accurate but I'm not a person who codes. I'm just a curious guy who loves Linux. 
Thx you for your messages.

**Reply by boredland** [reply]
I think that error means that there is something wrong with the drivers. Can you run `sudo mhwd -a pci nonfree 0300`, restart and try again?

**Reply by tinker-net** [reply]
I don't have logs, because It just everithing falls apart. But, While installing the screen go black and the monitor lost conection with de GPU. It stays black for a while so i restart and then It shows the first error. I've tried unsuported GPU, wich show the second error. 

I ve tried a tty to reinstall or uninstall anything works and my installation us ruined. I must reinstall (its ok, i asume we are testing things) 

For reinstalation i have try the Nvidia drivers, and the result is the same (un Sway the first error, in unsuported the second)

What can i do? IS there a way yo fix this? 
Thx a lot.

**Reply by tinker-net** [reply]
I know this is not the way It should be done, but It is the way i found.
The  photo is the Last log It shows before crash with the screen going black 
(In the live session It works equally)
![PXL_20230813_235840705](https://github.com/manjaro-sway/manjaro-sway/assets/141952222/39cc5b7a-b926-4c28-a07e-34507ab15419)

**Reply by tinker-net** [reply, marked as answer]
i've try anything i can (and have found, even with chatgpt help) to fix this with no results. At this point i'm sure that wayland or grub or something, surely you know better, doesn`t load my GPU or the Pascal family drivers are missed (but i don't think this can be an option because i insist in endeavour and garuda it works well and in i3 manjaro too. 
I'm quite sure that this is a bug in the sway .iso, related with nvidia drivers and, for sure the pascal family, i don't know if more families. 

I will be appreciate if this is fixed, but until that moment i can`t use this distro. So, if it will be soon, i wait, if not i install and use manjaro i3, 
Thank you for all your comments and support, it is really cool to have help.

**Reply by tinker-net** [reply]
Ok, my Bad. i3 Manjaro, does not work to. 
So... It seems It is a bug of Manjaro. 
I'm sorry. 
I Will redirect my issue to Manjaro forum.
Thank you so much!
I hope this issue can be fixed because i can use i3 instead of Sway but not be able to use any Manjaro It a shame.

**Reply by tinker-net** [reply]
In Manjaro Gnome, i managed to make my GPU (with Nvidia propietary) work, even with Vulkan.
